### DISCUSSION TITLE
CS00234 Error in Uno Wasm Project (The type or namespace name 'Install' does not exist in the namespace))

### DISCUSSION THREAD

**Discussion by inforithmics** [unoplatform/uno#9054] [Q&A]
When building an Uno Wasm project I get The type or namespace name 'Install' does not exist in the namespace 'System.Configuration' (are you missing an assembly reference?)

Project on Github
https://github.com/Mapsui/Mapsui

Solution:
Mapsui.sln

Project:
[Mapsui.Samples.Uno.Wasm](https://github.com/Mapsui/Mapsui/tree/master/Samples/Mapsui.Samples.Uno/Mapsui.Samples.Uno.Wasm)

The reason is that it creates an emdsk folder with the file that doesn't compile. On Github Actions it works only in Visual Studio 2022 Preview it fails.

The namespace that is missing there is a .NET 4.8 namespace which doesn't exist in .NET 6 so it seems there is somehow a Framework mixup.

**Comment by jeromelaban** [comment]
Thanks for the report. I tried building your latest master, but I'm not getting the error you're mentioning. Is there a specific branch to use? Otherwise, could you [provide a binlog](https://platform.uno/docs/articles/uno-builds-troubleshooting.html#generating-msbuild-binary-log-files)?

**Reply by inforithmics** [reply]
Thanks for analysing it. I tried it on a different Machine and there it worked, it seems on the machine where it fails I need to reinstall Visual Studio 2022 Preview.
